packages/exoflex/src/components/DateTimePicker/DateTimePicker.web.tsx

```diff
@@ -173,11 +206,21 @@ export function TimePickerContainer(props: PickerProps) {
         </View>
       </View>
       <View style={styles.touchableActionWrapper}>
-        <TouchableRipple onPress={onCancel} style={styles.touchableAction}>
-          <Text>CANCEL</Text>
+        <TouchableRipple
+          accessibilityHint="Close the picker"
```

Comment:
I thought we're going to expose the a11y hints, too?

Reply:
Oops, missed that one.